static/js/containers/ProfilePage.js

```diff
@@ -147,7 +148,11 @@ class ProfilePage extends React.Component {
       })
     ));
 
+    let text = `Welcome ${profile.preferred_name || SETTINGS.name}, let's
+    complete your enrollment to MIT Micromasters.`;
```

Comment:
Can you rename to `MicroMaster's` per #381? @pdpinch confirmed that we should have this spelling here